crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -92,8 +88,8 @@ pub async fn create(
     }
 
     Ok(AccountCreateResponse {
-        address: Hex(address),
-        max_fee: Decimal(max_fee),
+        address: Felt(address),
+        max_fee: Felt(max_fee),
```

Comment:
Does this change the default returned value format? (I mean does it print out as 0x1 instead of 1?)
I think we should print out max_fee as numbers, not hexes.

Reply:
Makes sense.

We currently apply number formatting to all of the output values. The way I see it, we can create a separate class e.g. `FeltAsDecimal`, or a allow creating a `Felt` while specifying it's default string format. I don't think we should use current `Decimal`, since u64 is not guaranteed to be enough for `max_fee`.

Reply:
I meant just by default. If someone passes --int, everything should be int, and likewise when someone uses --hex everything should be hex. But, if no explicit flag was passed, we should print out addresses/hashes as hex, and max_fee (potentially others, if they make sense) as numbers.

Reply:
I understand, I was rather asking about the approach that should be used 😅 
To be more memory efficient, decided to go for a separate `FeltAsDecimal` (not completely sure about the name).

Reply:
ok, I must've misunderstand you. 
I'm not sure having a separate struct, that is exaxctly the same as Felt struct, is a way to go. I think we should rather have multiple serializers instead, and use one or the other depeding on needs